test/ConfigTestCases.test.js

```diff
@@ -39,8 +40,9 @@ describe("ConfigTestCases", () => {
 					this.timeout(30000);
 					const testDirectory = path.join(casesPath, category.name, testName);
 					const outputDirectory = path.join(__dirname, "js", "config", category.name, testName);
-					const options = require(path.join(testDirectory, "webpack.config.js"));
-					const optionsArr = [].concat(options);
+					let options = require(path.join(testDirectory, "webpack.config.js"));
+					options = prepareOptions(options);
+					let optionsArr = [].concat(options);
```

Comment:
@sokra Done by wrapping the `require` directly in a call to `prepareOptions`, if that's cool.